testing/go/auth_test.go

```diff
@@ -15,9 +15,27 @@
 package _go
 
 import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
 	"testing"
 
 	"github.com/dolthub/go-mysql-server/sql"
+
+	"github.com/dolthub/doltgresql/server/functions"
+)
+
+const (
+	superUser = "doltgres_super"
+	superPass = "admin_password"
+	basicUser = "doltgres_basic"
+	basicPass = "user_password"
+)
+
+var (
+	createSuperUser = fmt.Sprintf("create user if not exists '%s' with superuser password '%s';", superUser, superPass)
+	createBasicUser = fmt.Sprintf("create user if not exists '%s' with password '%s'", basicUser, basicPass)
```

Comment:
I'd rename these to be specific to this file, something like `authTestSuperUser`, etc. We may want to have similarly named variables in other test files, so they would all need to be file-specific.